sensor_msgs/Image image
---
float32[]             eyes_closed
